CONTRIBUTING.md

```diff
@@ -1,33 +1,37 @@
+2. Test your changes made on a **physical** device. If you're adding or updating a package, test your changes with a build of the package you added/updated
+3. Create a Pull Request for your changes in your specific branch — include a description of the change and where you tested your change
+
+Don't expect your changes to be merged to pushed to upstream immediately — your changes will be reviewed by a team member (or Hayden), providing suggested changes which must be met.
+
+Once all further changes have been met (if any) and your changes are approved, they will be merged into upstream. The APT repository will shortly update showcasing your changes.
 
 ### Adding a New Package
-Adding new packages to Procursus is fairly easy and simple. For most packages, you're likely to need 2 files, only: a ``.mk`` file and a ``.control`` file. For documented examples, take a look at the [``grep.mk`` template](./grep.mk.template) and its [control file](./grep.control).
+Adding new packages to Procursus is fairly easy and simple. For most packages, you're likely to need 2 files
+- A ``.mk`` file, that builds the package, and..
+- A ``.control`` file, which contains info about your package
 
 Some things to keep in mind when adding a new package
-- Always ensure that the Architecture and Maintainer fields of your control file are populated with ``@DEB_ARCH@`` and ``@DEB_MAINTAINER@``, respectively
-- If you take advantage of already-made patchfiles from an external source, download them and implement them to your project. Check out it's done in [``bash.mk``](./bash.mk)
-- If the package you're adding only requires a few small edits, try taking advantage of ``sed`` in your packages' setup stage
+- Always ensure that the Architecture and Maintainer fields of your control file are populated with ``@DEB_ARCH`` and ``@DEB_MAINTAINER@``, respectively
+- If you take advantage of already-made patchfiles from an external source, download them and implement them to your project. Check out how it's done in [``bash.mk``](./makefiles/bash.mk)
+- If the package you're adding only requires a few small edits, try taking advantage of ``sed`` in your packages' setup stage, and..
 - Use designated functions found in the [documentation](https://github.com/ProcursusTeam/Procursus/wiki) to download, patch, or make other changes to your package
-- **Always** test your changes on a physical device with a build of the package you're attempting to add
 
 ### Updating a Package
 Package update contributions are always welcome, as not all team members are aware of new versions provided by the build-system. For this specific scenario, there's two options
 
 - **Fork and pull request**. Most of the time, updating packages is as simple as updating the version number in the ``.mk`` file of the package and recompiling. If you have the ability to test new versions of packages, feel free to go this route (following [package addition guidelines](#adding-a-new-package)). Otherwise...
```

Comment:
```suggestion
- **Fork and pull request**. Most of the time, updating packages is as simple as updating the version number in the `.mk` file of the package and recompiling. If you have the ability to test new versions of packages, feel free to go this route (following [package addition guidelines](#adding-a-new-package)). Otherwise...
```